apps/mux/MARKETPLACE.md

```diff
@@ -0,0 +1,39 @@
+A Contentful UI app that makes it simple to add beautiful streaming via [Mux](https://https://mux.com) to your [Contentful](https://contentful.com) project. Just install the extension, add the component to your content model, and you're good to go! 🙌🏾
```

Comment:
This file should be removed, it refers to the legacy extension.

Let's collaborate on communication (docs, marketplace entry) outside of this PR.